extensions/vscode/src/types/messages/hostToWebviewMessages.ts

```diff
@@ -20,7 +20,6 @@ export enum HostToWebviewMessageType {
   PUBLISH_FINISH_SUCCESS = "publishFinishSuccess",
   PUBLISH_FINISH_FAILURE = "publishFinishFailure",
   UPDATE_CONTENTRECORD_SELECTION = "updateContentRecordSelection",
-  UPDATE_CONFIG_SELECTION = "updateConfigSelection",
```

Comment:
We no longer update the config selection. The config is ALWAYS determined by what configuration name is present within the contentRecord, which is read from the file. If we want to change it, we patch the deployment, thus updating the file and causing a refresh, where the new configuration name is picked up.